Src/VimCore/Modes_Normal_NormalMode.fs

```diff
@@ -359,17 +356,8 @@ type internal NormalMode
         _data <- EmptyData
 
     member x.CanProcess (keyInput: KeyInput) =
-        if _runner.IsWaitingForMoreInput then 
-            true
-        elif _runner.DoesCommandStartWith keyInput then
-            true
-        elif Option.isSome keyInput.RawChar && VimKeyModifiers.None = keyInput.KeyModifiers then
-
-            // We can process any printable character (think international input)
-            // or any character which is part of the standard Vim input set.
-            CharUtil.IsPrintable keyInput.Char || Set.contains keyInput.Char _coreCharSet

```

Comment:
This old code always smelled a bit to me. I'm glad you've been able to so thoroughly identify the problem here and get a better solution.